dev-cpp/robin-map/robin-map-0.6.3.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="C++ fast hash map and hash set using robin hood hashing"
```

Comment:
Does this run the tests btw? (Not tried yet).

Reply:
No, its tests are very badly written.
Its a different project, in the test directory, with a new project name.
I would need to patch the cmake so that instead of a new project its added as a subdirectory and then fix the compile flags and other things.

That's very high cost to benefit for a package which installs just two header files...

Reply:
OK.

Reply:
I don't remember but I did try that, and it didnt work.
Lemme reconstruct the error.

Reply:
Hmm, it seems to be just ignoring it, not sure what its trying to do.
Here is what I'm trying - https://bsd.ac/5fn5gfh

Here is the link to testing directory - https://github.com/Tessil/robin-map/blob/master/tests/CMakeLists.txt
(I manually ran the tests and they worked)

I also feel like this has no benefit to testing.
Testing is mainly for checking if installation is working, which in this case is easy enough to see.
Any bug is going to be told to go upstream.
Unless someone wants to fix a bloom filter themselves...